packages/documentation/copy/en/reference/ESM Support for Node.md

```diff
@@ -1,23 +1,19 @@
 ---
-title: Experimental Support for ECMAScript Modules in Node.js
+title: ECMAScript Modules in Node.js
 layout: docs
 permalink: /docs/handbook/esm-node.html
-experimental: true
 oneline: Using ECMAScript Modules in Node.js
 ---
 
-> Note: this functionality is currently experimental and prone to changing.
-> It can currently only be used on [nightly builds](/docs/handbook/nightly-builds.html) of TypeScript,
-> and we are currently seeking feedback around bugs and pain-points.
-
 For the last few years, Node.js has been working to support running ECMAScript modules (ESM).
-This has been a very difficult feature to support, since the foundation of the Node.js ecosystem is built on a different module system called CommonJS (CJS).
-Interoperating between the two brings large challenges, with many new features to juggle;
-however, support for ESM in Node.js is now largely implemented in Node.js 12 and later, and the dust has begun to settle.
+This has been a vesry difficult feature to support, since the foundation of the Node.js ecosystem is built on a different module system called CommonJS (CJS).
```

Comment:
Noticed a typo here from the Twitter thread! 😅  `vesry` -> `very`

Reply:
Thanks!